docs/tutorial/07-batch-infer.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
     "# Batch Inference and Drift Detection\n",
     "\n",
-    "In this tutorial, we will be leveraging a function from the [MLRun Function Marketplace](https://www.mlrun.org/marketplace/) to perform [batch prediction](https://github.com/mlrun/functions/tree/development/batch_predict) using a logged model and new prediction dataset. The function will also calculate data drift by comparing the new prediction dataset with the original training set.\n",
+    "In this tutorial, we will be leveraging a function from the [MLRun Function Marketplace](https://www.mlrun.org/marketplace/) to perform [batch inference](https://github.com/mlrun/functions/tree/master/batch_inference) using a logged model and new prediction dataset. The function will also calculate data drift by comparing the new prediction dataset with the original training set.\n",
```

Comment:
This tutorial leverages a function from the [MLRun Function Marketplace](https://www.mlrun.org/marketplace/) to perform [batch inference](https://github.com/mlrun/functions/tree/master/batch_inference) using a logged model and new prediction dataset. The function also calculates data drift by comparing the new prediction dataset with the original training set.\n",